### ISSUE TITLE
Download files in Android / Cordova

### ISSUE THREAD

**Issue by Luvelnet** [veliovgroup/Meteor-Files#835]
Hello:

In my meteor App I have a document download section, which works perfectly from the browser. But when I try to download a file from Android directly it doesn't do anything.

I have read that this package is only dedicated to uploading files but not downloading from Cordova, I don't know if I'm right, if so, can you guide me how to download?

Use meteor 2.7.1

Thank you very much

**Comment by dr-dimitru** [comment]
@Luvelnet do you get an error? Have you tried adding [`?download` option](https://github.com/veliovgroup/Meteor-Files/blob/7f67a0885e120ad77085b191e3399add575a2d89/docs/template-helper.md#get-download-url-for-file-pass-fileref-object-to-helper)?

**Comment by dr-dimitru** [comment]
@Luvelnet Feel free to close it in case if the issue is solved on your end.

**Comment by Luvelnet** [comment]
Yes I solved the issue, but not with download option. For me it was necesary install the Cordova Plugin File Transfer [https://github.com/apache/cordova-plugin-file-transfer](url). I hope it will help someone. 

Thanks!
